File: NotificaCrimesBackEnd/AppNotificacoesCrimesCidade.Domain/Entities/LocalizacaoOcorrencia.cs
using NetTopologySuite.Geometries;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppNotificacoesCrimesCidade.Domain.Entities
{
    public class LocalizacaoOcorrencia
    {
        [Column("id")]
        public int Id { get; set; }

        [Column("cep")]
        public string? cep { get; set; }

        [Column("coordenadas")]
        public Point Coordenadas { get; set; }

        [Column("cidade")]
        public string? Cidade { get; set; }

        [Column("bairro")]
        public string? Bairro { get; set; }

        [Column("rua")]
        public string? Rua { get; set; }

        [Column("numero")]
        public int Numero { get; set; }
    }
}

File: notifica_crimes_frontend/lib/domain/models/ocorrencias/localizacao_ocorrencia.dart
class LocalizacaoOcorrencia {
  final String? cep;
  final double latitude;
  final double longitude;
  final String? cidade;
  final String? bairro;
  final String? rua;
  final int? numero;

  LocalizacaoOcorrencia({required this.cep, required this.latitude, required this.longitude, required this.cidade, required this.bairro, required this.rua, required this.numero});
}